packages/fxa-admin-server/package.json

```diff
@@ -18,7 +18,7 @@
     "delete": "pm2 delete pm2.config.js",
     "test": "yarn gen-keys && yarn test-default && yarn test-e2e ",
     "gen-keys": "node -r esbuild-register ./src/scripts/gen_keys.ts;",
-    "test-unit": "JEST_JUNIT_OUTPUT_FILE=../../artifacts/tests/$npm_package_name/jest-unit.xml jest --runInBand --coverage --forceExit --logHeapUsage -t '^(?!.*?#integration).*' --ci --reporters=default --reporters=jest-junit",
+    "test-unit": "JEST_JUNIT_OUTPUT_FILE=../../artifacts/tests/$npm_package_name/jest-unit.xml jest --runInBand --coverage --detectOpenHandles --forceExit --logHeapUsage -t '^(?!.*?#integration).*' --ci --reporters=default --reporters=jest-junit",
```

Comment:
It threw an error on force exit and suggested adding detect open handles.... and now it says it all passed? 🤷🏻

Reply:
Should we be concerned about this note regarding [detectOpenHandles](https://jestjs.io/docs/cli#--detectopenhandles)?

> This option has a significant performance penalty and should only be used for debugging.